hack/actions/install-kubernetes-toolchain.sh

```diff
@@ -10,6 +10,10 @@ readonly KIND_VERS="v0.11.1"
 readonly SONOBUOY_VERS="0.19.0"
 readonly KUBEBUILDER_VERS="3.1.0"
 
+# The CI env variable is set by Github Actions when running jobs.
```

Comment:
this section seems removable?

Reply:
I think I had that since I was running this script locally to install some deps. But we should approach that a different way I think.